e2e/specs/appointments.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, type Patient, startVisit, endVisit } from '../commands';
+import { test } from '../core';
+import { AppointmentsPage } from '../pages';
+import { type Visit } from '@openmrs/esm-framework';
+
+let patient: Patient;
+let visit: Visit;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+  visit = await startVisit(api, patient.uuid);
+});
+
+test('Add, edit and cancel an appointment', async ({ page, api }) => {
+  const appointmentsPage = new AppointmentsPage(page);
+
+  await test.step('When I go to the appointment tab in the patient chart', async () => {
+    await appointmentsPage.goto(patient.uuid);
+  });
+
+  await test.step('And I click on the “Add” button', async () => {
+    await page.getByRole('button', { name: 'Add', exact: true }).click();
+  });
+
+  await test.step('And I select Mobile Clinic location', async () => {
+    await page.getByLabel(/select location/i).selectOption('Mobile Clinic');
+  });
```

Comment:
The test is failing because we don't have the data for location field in dev 3 instance as per this [video](https://github.com/openmrs/openmrs-esm-patient-management/pull/1062#issuecomment-2028835280)